lib/azure_text_to_speech.ex

```diff
@@ -17,18 +17,22 @@ defmodule AzureTextToSpeech do
     audio-48khz-96kbitrate-mono-mp3     audio-48khz-192kbitrate-mono-mp3)
   @user_agent "awesome"
 
+  @spec voices_list :: map()
   def voices_list do
     voices_list_endpoint()
     |> HTTPoison.get!(authorization_header(access_token()))
     |> Map.get(:body)
     |> Jason.decode!()
   end
 
+  @spec audio_list :: [bitstring(), ...]
```

Comment:
I would standardize on `binary` type (or `String.t()` type). My understanding is that `bitstring` type is looser than `binary` and is useful when a number of bits is not divisible by 8. In our case, `binary` works.

```suggestion
  @spec audio_list :: [binary]
```